Gemfile

```diff
@@ -13,7 +13,11 @@ group :test do
   gem 'httpclient', '>= 2.2'
   gem 'mime-types', '~> 1.25', :platforms => [:jruby, :ruby_18]
   gem 'minitest', '>= 5.0.5'
-  gem 'net-http-persistent', '~> 2.9.4'
+  if RUBY_VERSION >= '2.1'
```

Comment:
No, I personally think that even though it's not the most elegant solution, this is the only one we can use in this specific case. So I'm saying that for me it's ok as it is.